### PR TITLE
Apply blockutils

### PR DESCRIPTION
**Pull request by markusuwe**
Apply new blockutils package and remove local instances of same functionality.

Got a little bit more complicated than expected as the superresolution block does some special things, but in the end it was not too much work and it removes quite a but of code.

Unrelated I added and install option without GPUs.

### CODE REVIEW

File: src/s2_tiles_supres.py [hunk 1]
```diff
@@ -287,26 +294,21 @@ def data_final(
                 )[:, :, term]
         return d_final
 
-    # pylint: disable-msg=too-many-locals
-    # pylint: disable-msg=too-many-statements
-    def run_model(self):
+    # pylint: disable-msg=arguments-differ
```
Comment (markusuwe) [inline src/s2_tiles_supres.py:297]: Couldn't get rid of this. Any ideas anybody?

**Comment by nekhtiari** [comment]
Nice, many changes.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,11 @@ Install the necessary Python libraries via:
 make install
 ```
 
+Or if you are running on an environmeant that does not support GPUs
+```bash
+make install-cpu
+```
+
```
Comment (rodrigoalmeida94) [inline README.md:71]: Nice!

File: src/s2_tiles_supres.py (hunk 1 above)
Comment (rodrigoalmeida94) [reply src/s2_tiles_supres.py:297]: weird... I guess you are using the latest blockutils? version 0.0.6?

File: tests/test_s2_tiles_supres.py [hunk 3]
```diff
@@ -216,10 +236,26 @@ def test_data_final():
...
+    Checks if class method from_dict of ProcessingBlock works
+    """
+
+    params = {
+        "bbox": [12.211, 52.291, 12.513, 52.521],
+        "clip_to_aoi": True,
+        "copy_original_bands": False,
+    }
+    supres = Superresolution.from_dict(params)
+    assert isinstance(supres, Superresolution)
```
Comment (rodrigoalmeida94) [inline tests/test_s2_tiles_supres.py:261]: Nice one!

File: src/s2_tiles_supres.py (hunk 1 above)
Comment (markusuwe) [reply src/s2_tiles_supres.py]: With version 0.0.6 it got resolved. Thanks.

**Pull request merged**